cmd/virt-launcher/virt-launcher.go

```diff
@@ -465,14 +465,14 @@ func main() {
 
 	domain := waitForDomainUUID(*qemuTimeout, events, signalStopChan, domainManager)
 	if domain != nil {
-		var pidDir string
+		var pidDirArray []string
 		if *runWithNonRoot {
-			pidDir = "/run/libvirt/qemu/run"
+			pidDirArray = []string{"/run/libvirt/qemu/run", "/var/run/libvirt/qemu/run"}
 		} else {
-			pidDir = "/run/libvirt/qemu"
+			pidDirArray = []string{"/run/libvirt/qemu", "/var/run/libvirt/qemu"}
```

Comment:
It has been a while and at least one vacation since i did that change, but the reason was because virt-launcher failed to find the PID file for the process. This caused it to kill the VM after 5 min. When inspecting the container, i noticed that the PID file did exist, but was not in the right folder. So i did this change to make it look into multiple folders instead.

`/run` and `/var/run` should be the same, but somehow on the s390x image they are not. They are also not a symlink...

Reply:
Could be that bazel didn't created that symlink in the s390x image. Please, try to double check in the BUILD.bazel, we have examples where we create symlinks.

Reply:
Yes, that's exactly it! For aarch64 we have

```
rpmtree(
    name = "launcherbase_aarch64",
    capabilities = {
        "/usr/libexec/qemu-kvm": [
            "cap_net_bind_service",
        ],
    },  
    rpms = [
        ...
    ],
    symlinks = {
        "/etc/crypto-policies/back-ends/openssl.config": "/usr/share/crypto-policies/DEFAULT/openssl.txt",
        "/etc/crypto-policies/back-ends/opensslcnf.config": "/usr/share/crypto-policies/DEFAULT/opensslcnf.txt",
        "/var/run": "../run",
        "/usr/sbin/iptables": "/usr/sbin/iptables-legacy",
        "/usr/bin/nc": "/usr/bin/ncat",
    },
    visibility = ["//visibility:public"],
)
```

but s390x is missing both the `capabilities` and the `symlinks` bits.

Please make sure the various `rpmtree`s are defined fully and drop the unnecessary code changes.

Reply:
Fixed with latest update. I have removed the unnecessary code.